bluebottle/tasks/serializers.py

```diff
@@ -146,6 +150,24 @@ class Meta:
             'type',
         )
 
+    def _check_project_deadline(self, instance, validated_data):
+        project = validated_data['project']
+        if instance.deadline > project.deadline:
+            project.deadline = instance.deadline
+            project.save()
```

Comment:
I think this should only update if it's a sourcing project and not for funding (or combined) projects.